homeassistant/components/devolo_home_control/devolo_device.py

```diff
@@ -0,0 +1,83 @@
+"""Base class for a device entity integrated in devolo Home Control."""
+import logging
+
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN
+from .subscriber import Subscriber
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_BATTERY_LEVEL = "battery_level"
+
+
+class DevoloDeviceEntity(Entity):
+    """Representation of a sensor within devolo Home Control."""
+
+    def __init__(self, homecontrol, device_instance, element_uid, name, sync):
+        """Initialize a devolo device entity."""
+        self._device_instance = device_instance
+        self._name = name
+        self._unique_id = element_uid
+        self._homecontrol = homecontrol
+        self._available = device_instance.is_online()
```

Comment:
Is this doing i/o?

Reply:
No.
https://github.com/2Fake/devolo_home_control_api/blob/ddc208537e02062f64490245e394511ff15e0f9d/devolo_home_control_api/devices/zwave.py#L57

Reply:
Generally we make these properties to indicate no I/O is happening.  Its fine for now.

Reply:
Please add a comment about this jot  doing I/O in the code. I've asked the same thing in a previous PR, I think.

Reply:
Done in https://github.com/home-assistant/core/pull/36557